src/js/components/inspectors/Property.jsx

```diff
@@ -5,34 +5,26 @@ var React = require('react'),
     ContentEditable = require('../ContentEditable'),
     model = require('../../model'),
     lookup = model.lookup,
-    resetProperty = require('../../actions/ruleActions').resetProperty,
-    vegaInvalidate = require('../../actions/vegaInvalidate'),
-    addVegaReparseRequest = require('../mixins/addVegaReparseRequest');
```

Comment:
This is only tangentially related to this PR; individual actions that would invalidate Vega (adding or removing marks, binding data, etc) handle the invalidation themselves, so the explicit invalidator call is not needed here.